src/lib/Repository/DecoratedLocationRepository.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace EzSystems\HybridPlatformUi\Repository;
+
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use eZ\Publish\API\Repository\Values\Content\Location;
+use EzSystems\HybridPlatformUi\Decorator\LocationDecorator;
+
+class DecoratedLocationRepository
```

Comment:
DecoratedLocationService, not Repository. Repository has another meaning in the eZ Platform domain. Please also update the service name.